djangocms_admin_style/sass/components/_forms.scss

```diff
@@ -792,6 +811,10 @@ form select[multiple] {
         width: calc(100% - 91px) !important;
         margin-right: 10px;
     }
+    select + .related-widget-wrapper-link,
+    .related-widget-wrapper-link + .related-widget-wrapper-link {
```

Comment:
as I understand we do not have influence on html, because such selector are too heave and looks weird a bit